core/survey/nodeDefLayout.js

```diff
@@ -80,6 +80,14 @@ export const dissocLayoutChildren = (cycle) => R.dissocPath([cycle, keys.layoutC
 
 export const assocPageUuid = (cycle, pageUuid) => assocLayoutProp(cycle, keys.pageUuid, pageUuid)
 
+export const copyLayoutSurveyCycle = (surveyCycleKeyFrom, surveyCycleKeysNodeDef) => (nodeDef) => {
```

Comment:
let's start using object parameters and shortening the names. this can be
```export copyLayout = ({cycleFrom, cyclesTo}) => (nodeDef)... ```

Reply:
ok